site/content/docs/5.1/components/orange-local-headers.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: docs
+title: Orange local headers
+description: Documentation and examples for Boosted's exclusive Brand responsive title bars, breadcrumbs and local navigation bars.
+group: components
+toc: true
+---
+
+## Examples
```

Comment:
I don't understand why there is an "Examples" part an then a "Title bars" with almost the same content. IMO it should be merged into one: "Title bars".